shell/platform/darwin/macos/framework/Source/FlutterSurfaceManager.mm

```diff
@@ -0,0 +1,128 @@
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterSurfaceManager.h"
+#import "flutter/fml/logging.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/MacOSSwitchableGLContext.h"
+
+#include <OpenGL/gl.h>
+
+enum {
```

Comment:
Is this necessary in this instance? The values are used as array indices and enum class do not implicitely convert to integers.

Reply:
Ah, in that case please just fix the names so they aren't extremely generic (and thus prone to collisions with other enums; this may be private, but if a public enum were ever added visible from this file with the same names, it would be a mess).